File: Search/app/src/main/java/com/example/rough/ResponseViewHolder.java
package com.example.rough;

import android.view.View;
import android.widget.ImageView;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

import com.bumptech.glide.Glide;

public class ResponseViewHolder extends RecyclerView.ViewHolder {

    ImageView mImageView;

    public ResponseViewHolder(@NonNull View itemView) {
        super(itemView);
        mImageView = itemView.findViewById(R.id.ivImage);
    }

    public void setData(ResultsItem resultsItem) {
        Glide.with(mImageView).load("https://image.tmdb.org/t/p/w500/" + resultsItem.getPosterPath()).into(mImageView);
    }
}

File: 8.SearchFunctanility/app/src/main/java/com/example/searchfuncationality/ResponseViewHolder.kt
package com.example.searchfuncationality

import android.view.View
import androidx.recyclerview.widget.RecyclerView
import com.example.searchfuncationality.Model.DataModel
import kotlinx.android.synthetic.main.item_layout.view.*

class ResponseViewHolder (itemView: View) :
    RecyclerView.ViewHolder(itemView) {

    fun setData(dataModel:DataModel) {
        itemView.apply {
            dataModel.run {
               tvName.text = dataModel.movieName
            }
        }
    }
}